plugins/cinema4d/import_cast.pyp

```diff
@@ -366,21 +374,21 @@ def importSkeletonIKNode(doc, modelNull, skeleton, boneIndexes):
         ikTag[c4d.ID_CA_IK_TAG_SOLVER] = 2
         ikTag[c4d.ID_CA_IK_TAG_TIP] = endBone
         ikTag[c4d.ID_CA_IK_TAG_TARGET] = ikTargetNull
-        
+
         poleVectorBone = handle.PoleVectorBone()
         if poleVectorBone is not None:
             poleVector = boneIndexes[poleVectorBone.Hash()]
-            ikPoleNull = BaseObject(c4d.Onull)
-            ikPoleNull.SetName(poleVector.GetName() + "_Pole")
-            ikPoleNull.SetMg(poleVector.GetMg())
-            ikPoleNull.SetAbsRot(Vector(0, 0, 0))
-            
-            doc.InsertObject(ikPoleNull, startBone)
-            
             ikTag[c4d.ID_CA_IK_TAG_POLE] = poleVector
+            ikTag[c4d.ID_CA_IK_TAG_POLE_AXIS] = POLE_AXIS_X
```

Comment:
Doesn't c4d have this as a global constant somewhere?

Reply:
You are right, I thought it was just an integer